_sass/_layout.scss

```diff
@@ -372,3 +372,32 @@ q:after {
 .downloads-content p:first-child{
   color: red;
 }
+
```

Comment:
It would be nice to create separate scss files for each type of content. If not this file will be come huge in no time.

Reply:
There is not much more scss to add into _layout.scss, i should create another scss?

Reply:
Yes, let's try not to add everything to this file.

Reply:
Maybe i should create it when is merged to gh-pages? Maybe i have more similar lines in the same scss

Reply:
Sure, I just approved and merged SH-3. So you can fix the conflicts and create the other file.